### DISCUSSION TITLE
Can we use Conversation without the global indexing ?

### DISCUSSION THREAD

**Discussion by mdouchin** [protz/thunderbird-conversations#1849] [General]
Hi,

I encoutered very bad performances in Thunderbird with an INBOX folder containing 25000 messages. Since I never use the full text search (global search) in Thunderbird, I decided to disable it: before, TB was stuck any time the index was refreshed (in the Activities dialog) and could not load the message.

I know that the conversation module asks to activate the indexing during the first use wizard.

I would like to know if it is mandatory, or only recommended ?

NB: I activated the auto-deletion of messages older than 5 years, but 14000 still remain, and it seems it is too big for the TB indexing tool. I could auto-archive old messages to remove them from my Inbox, but would prefer to keep things as is if possible

**Comment by Standard8** [comment]
Conversations is pretty much useless without the global index. You would loose the functionality of related messages being displayed in the message pane at the same time - Conversations needs the global index to make that work. I suspect some other functionality would be disabled as well (like recent conversations for a contact). 

Without it, Conversations would just be an alternate display for the message pane which is not really its point.

Thunderbird will take time to index all the messages when it first is enabled, this process can take many hours. In theory it should be usable during this process, but you can always try waiting for it to end, the activity manager under the tools menu will give you some idea of what's happening.

If it still isn't usable after the indexing is complete, then this would be an issue to raise with Thunderbird itself at https://support.mozilla.org/products/thunderbird

**Reply by mdouchin** [reply]
Thanks a lot for your answer. I will try to reactivate the indexing and be more patient until it completes.
